Homework_7_Java_Arrays_Loops/JavaArraysAndLoops/src/StrongestRobot.java

```diff
@@ -0,0 +1,24 @@
+public class StrongestRobot {
+
+    public static void main(String[] args) {
+
+        int[] weights = {300, 450, 200, 500, 350}; // 1-ий робот підняв 300кг, другий 450 і т.п.
+        String[] robotName = {"IronBolt", "MuscleBot", "PowerMax", "SteelStorm", "TitanLift"};
+
+        int MaxWeight = Integer.MIN_VALUE;
```

Comment:
The only thing: variable naming

This is a non-static local variable, so should be `maxWeight` (lower camel case according to convention)